src/views/__tests__/XdrViewer.test.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * @jest-environment jsdom
+ */
+
```

Comment:
I had been holding off on merging because I wanted to add more tests, but I can do that in a later PR